app/src/back-office/views/layouts/home-layout.jsx

```diff
@@ -21,6 +31,36 @@ export default class HomeLayout extends Component {
 
     togglePopover = (e) => this.setState({open: !this.state.open, anchorEl: e.currentTarget});
 
+    async saveArticle() {
+        const title = this.refs.title.getValue();
+        const description = this.refs.description.getValue();
+        if (title && description) {
+            try {
+                const id = await this.props.saveArticle(title, description);
+                this.props.showSnackBar({
+                    message: 'edit-cartridge.content.snackBar.saveSuccessMessage',
+                    isError: false
+                });
+                this.toggleModal();
+                setTimeout(() => this.props.router.push(`edit-article/${id}`), 500);
+            } catch (e) {
+                this.props.showSnackBar({
+                    message: 'edit-cartridge.content.snackBar.saveFailedMessage',
+                    isError: true
+                });
+            }
+        } else {
+            this.props.showSnackBar({
+                message: 'edit-cartridge.content.snackBar.saveFailedContentMessage',
+                isError: true
+            });
+        }
+    }
```

Comment:
I've just done the exact same as the first layout you did. Now we have three diffrent layouts (one for the home, another one for the consult and one for the edit mode). If you have an idea to make it in common i'll follow you for the snackbar ;) @JabX